ecl/hql/hqlexpr.ipp

```diff
@@ -988,15 +988,15 @@ typedef class MapXToMyClassViaBase<IAtom *, IAtom *, IFileContents, IFileContent
 
 class CHqlDelayedCall: public CHqlExpressionWithType
 {
-    OwnedHqlExpr param;
+    OwnedHqlExpr funcdef;
 protected:
     CHqlDelayedCall(IHqlExpression * _param, ITypeInfo * type, HqlExprArray &parms);
```

Comment:
@ghalliday Should the first argument be named _funcdef?  Also, note that CHqlDelayedCall drerived classes calls CHqlDelayedCall constructor with first parameter named _param - should these be renamed too?